lib/components/narrative/default/default-itinerary.js

```diff
@@ -31,6 +32,7 @@ export default class DefaultItinerary extends NarrativeItinerary {
           <span className='title'>Itinerary {index + 1}</span>{' '}
           <span className='duration pull-right'>{formatDuration(itinerary.duration)}</span>{' '}
           <span className='arrivalTime'>{formatTime(itinerary.startTime)}—{formatTime(itinerary.endTime)}</span>
+          <span className='pull-right'><LinkButton to='/savetrip'>Save</LinkButton></span>{' '}
```

Comment:
Once a trip is saved, there is no indication that the trip (itinerary) has been saved. It would be nice to have the itinerary display change accordingly once it is saved.

Reply:
That's a good point... I am not sure how to accomplish that without doing a full leg-by-leg comparison.

Reply:
Creating #209 to track this.